tools/skilldbconverter.php

```diff
@@ -654,7 +654,7 @@ function getnk($nk)
 {
 	$bitmask = array(
 		"NoDamage"         =>   intval(0x01), //- No damage skill
-		"SplashArea"       =>   intval(0x02)+intval(0x04), //- Has splash area (requires source modification)
```

Comment:
This in source is 
`NK_SPLASH         = 0x02|0x04, // 0x4 = splash & split`
therefore adding both options to the file as -
` 
	DamageType: {
		SplashArea: true
		SplitDamage: true
	}
`
is pointless.